### ISSUE TITLE
API getleds / setleds indexing mismatch

### ISSUE THREAD

**Issue by RaptDept** [psieg/Lightpack#387]
Given `n` number of lights, `getleds` returns lights starting from `0-r,g,b;1-r,g,b;...<n-1>-r,g,b;`.
Meanwhile, `setleds` seems to assume a format of `1-r,g,b;2-r,g,b;...<n>-r,g,b` and passes those numbers as-is.

I noticed this when I called `setleds` with the value returned by `getleds` multiple times, and it removed the first light and ignore the last one each time.

Digging into the source, it seems `LightpackPluginInterface` addresses lights with zero-based indexing, so I think the bug is in `setleds` assuming one-based indexing.

**Comment by RaptDept** [comment]
My bad, I read the code again, and it doesn't seem to matter what the index passed to `LightpackPluginInterface::SetLeds` is. Though, `ApiServer`'s `setleds` and `getleds` do have different indexing, so that kind of confused me.

**Comment by psieg** [comment]
Yes I think I once stumbled over that too. Unfortunately that's how the API was when I forked and I'd rather not break compatibility with existing clients...
